src/v3/src/types/schema.ts

```diff
@@ -311,7 +311,8 @@ export interface ImageWithTextElement extends UISchemaElement {
   options: {
     id: string;
     SVGIcon: FunctionComponent;
-    textContent: string;
+    textContent?: string;
+    alignment?: string;
```

Comment:
We should consider updating this in a future PR to use a string enum

Reply:
It needs more TLC any way, the name "alignment" is  misleading since the display is "flex"